### ISSUE TITLE
'Flag review' menu has layout issues when changing screen sizes

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-frontend#3843]
Steps to reproduce:

Prerequisites: Android device in landscape mode

1.Go to an add-on detail page and click on 'Read all reviews' (i.e. https://addons-dev.allizom.org/en-US/firefox/addon/bluhell-firewall/reviews/)
2. Pick one of the listed reviews and tap on 'Flag'
3. With the 'Flag' menu opened, rotate your device to portrait mode

Expected result:
The 'Flag' menu is re-positioned to fit to the new screen layout

Actual result;
The 'Flag' menu is cut on the right side

Notes: reproduced on all AMO servers with FF56 Android 7.0. 
I was not able to reproduce with RTL
This also happens on Desktop on smaller screen sizes

![flags](https://user-images.githubusercontent.com/31961530/32555770-51c50a1c-c4a6-11e7-8555-d1cda1ef58f5.gif)

![image](https://user-images.githubusercontent.com/31961530/32555847-7f00be7c-c4a6-11e7-8853-188f211221e3.png)

**Comment by kumar303** [comment]
> With the 'Flag' menu opened, rotate your device to portrait mode

I suppose it's fixable but this seems like an unrealistic thing for a user to do. They would probably begin the flagging process either in potrait mode or in landscape mode.

**Comment by tofumatt** [comment]
I don’t mean to be difficult but I definitely do this (switch from one orientation to another) a fair bit on my phone. I think it’s lower priority (especially as we have way more desktop users than mobile) but I think it’s worth fixing.

- Matt (Sent from mobile)

**Comment by kumar303** [comment]
I switch orientation too but switching while I'm in the middle of flagging something? That seems unrealistic. I guess you could do it by accident which does happen. Anyway, there is probably an easy way to fix it.

**Comment by rebmullin** [comment]
never mind...I was looking at the emulator and not sure hover is really a solution for mobile 

will look into this more..

**Comment by codeEmpress1** [comment]
Hi, I am an outreachy applicant and I will like to work on this

**Comment by willdurand** [comment]
> Hi, I am an outreachy applicant and I will like to work on this

@codeEmpress1 thanks for your interest in this project. 
You can find good first issues by looking at the [opened `contrib: welcome` issues](https://github.com/mozilla/addons-frontend/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22contrib%3A+welcome%22). This one is a bit tricky I think.

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDFRNT-125
